src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisController.java

```diff
@@ -203,4 +210,79 @@ public ResponseEntity<List<String>> getProviders() {
     public ResponseEntity<String> getDefaultProvider() {
         return ResponseEntity.ok().body(service.getDefaultProvider());
     }
+
+    @PostMapping(value = "/networks/{networkUuid}/non-evacuated-energy", produces = APPLICATION_JSON_VALUE, consumes = APPLICATION_JSON_VALUE)
```

Comment:
/networks/{networkUuid}/non-evacuated-energy/run-and-save ?

Reply:
Changed